libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/EmptyBinder.kt

```diff
@@ -7,17 +7,12 @@ import com.mapbox.navigation.core.MapboxNavigation
 import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
 import com.mapbox.navigation.dropin.R
 
-/**
- * This does not have a transition specified. To have better control of
- * the transition, create a new binder with your specified transition.
- * You can reuse the R.layout.mapbox_layout_drop_in_empty
- */
 @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
 class EmptyBinder : UIBinder {
```

Comment:
Should this be internal?

```suggestion
internal class EmptyBinder : UIBinder {
```

I think we're actually working to delete it, but maybe it is still needed.

Reply:
I made it internal but it is being used in `qa-test-app`. Had to revert it.